packages/react-accordion/src/common/isConformant.ts

```diff
@@ -6,6 +6,7 @@ export function isConformant<TProps = {}>(
   const defaultOptions: Partial<IsConformantOptions<TProps>> = {
     asPropHandlesRef: true,
     componentPath: module!.parent!.filename.replace('.test', ''),
+    skipAsPropTests: true,
```

Comment:
Can you please clarify why it is required?

Reply:
The `as` prop test should be reformulated. it's testing inappropriate  cases such as Function component, which is not supported anymore.

Reply:
Can you please create an issue to follow up?